includes/class-sensei-templates.php

```diff
@@ -567,19 +567,6 @@ public  static function deprecate_lesson_image_hook(){
     }// end sensei_deprecate_lesson_single_main_content_hook
 
     /**
-     * hook in the deprecated sensei_login_form hook for backwards
-     * compatibility
-     *
-     * @since 1.9.0
-     * @deprecated since 1.9.0
-     */
-    public static function deprecate_sensei_login_form_hook(){
```

Comment:
Related to my comment above:
> Yes. I'd say this change in general is a breaking change in those cases, so we probably want to document it as such.

I don't think it makes sense to deprecate them, since we removed the login form so it wouldn't make sense to keep them?